src/installerquick.h

```diff
@@ -32,9 +33,11 @@ class InstallerQuick : public MOBase::IPluginInstallerSimple
 
 private:
 
-  bool isSimpleArchiveTopLayer(std::shared_ptr<const MOBase::IFileTree> tree) const;
-  bool isDataTextArchiveTopLayer(std::shared_ptr<const MOBase::IFileTree>) const;
-  std::shared_ptr<const MOBase::IFileTree> getSimpleArchiveBase(std::shared_ptr<const MOBase::IFileTree> dataTree) const;
+  bool isDataTextArchiveTopLayer(
+    std::shared_ptr<const MOBase::IFileTree>, QString const& dataFolderName, ModDataChecker* checker) const;
+  
+  std::shared_ptr<const MOBase::IFileTree> getSimpleArchiveBase(

```

Comment:
Maybe a comment is in order to specify what this function does since it's not immediate from the name like it was before.

Reply:
I'm not sure it was clear before :D I'll move the comment inside to the declaration!